FundacionParaguaya/MentorApp
# Logout confirmation

This pull request aims to introduce a second step for logging out when there are unsynced stuff on the device. This doesn't cover removing cached images.

<img width="854" alt="68747470733a2f2f696d616765732e7a656e68756275736572636f6e74656e742e636f6d2f3563313966666139613839366461356438336436613064302f66346633373936612d326237612d343565322d396337362d323934373031633139396535" src="https://user-images.githubusercontent.com/3336245/51836245-8b94b500-2308-11e9-8310-af479a799d4f.png">

**To Test**
1. Login to any account
2. Make a single draft so there is unsynced stuff. You can even fill in just the first name and then use the dev menu to refresh instead of filling each required field.
3. Open the Log out modal from the side menu. You should see the same modal as usual.
4. Click the red Yes. Now the modal should show a list of checkboxes to be clicked.
5. Click on just one of them and then click Delete. The rest of the checkbox labels should underline and become red
6. Check all the boxes and click Delete. Not the user should be logged out as usual.
